src/components/VmsPageHeader/UserMenu.js

```diff
@@ -28,7 +28,7 @@ const UserMenu = ({ config, onLogout, onAdministratorFilterClick }) => {
   } else {
     let checkbox = null
     if (config.get('administrator')) {
-      checkbox = (<li><a><Checkbox checked={config.get('filter')} onClick={onAdministratorFilterClick} label='Filter VM' /></a></li>)
+      checkbox = (<li><a><Checkbox checked={config.get('filter')} onClick={onAdministratorFilterClick} label='Use Admin Permissions' /></a></li>)
```

Comment:
Maybe even better would be: "Show all entities" and having an info icon or something explaining what does it actually mean.
Also please keep in mind that the "Filter VM" and "Show All Entities" is inverse so you need to invert the value.